api/src/main/java/org/openmrs/module/attachments/AttachmentsServiceImpl.java

```diff
@@ -32,6 +33,10 @@
 	@Qualifier(AttachmentsConstants.COMPONENT_ATT_CONTEXT)
 	private AttachmentsContext ctx;
 	
+	@Autowired
+	@Qualifier(AttachmentsConstants.COMPONENT_COMPLEXDATA_HELPER)
+	private ComplexDataHelper complexDataHelper;
```

Comment:
Again, this shouldn't be needed when `AttachmentsContext` is injected, since there is a getter to it.